packages/cli/src/webhooks/__tests__/webhook-helpers.test.ts

```diff
@@ -449,3 +452,17 @@ describe('prepareExecutionData', () => {
 		expect(runExecutionData.resultData.pinData).toBeUndefined();
 	});
 });
+
+describe('getWebhookErrorMessage', () => {
+	const workflowStartNode = mock<INode>({ name: 'Start' });
+	it('should surface WorkflowConfigurationError', () => {
+		const err = new WorkflowConfigurationError(workflowStartNode, new Error('test'));
+		expect(_privateGetWebhookErrorMessage(err, 'Webhook')).toEqual(err.message);
+	});
+	it('should obfuscate other errors', () => {
```

Comment:
nit:
```suggestion
	});
	
	it('should obfuscate other errors', () => {
```

(should add a lint rule for this)